ChangeLog.md

```diff
@@ -0,0 +1,923 @@
+Changelog
+=========
+
+Version 3.1.0
+-------------
+27/07/2020
+
+ - General changes:
+
+   - Full support of CPACS 3.1
+   - Backport support for control surface devices from TiGL 2. It is now possible to manipulate trailing edge devices
+     with TiGL 3.1 using the TiGL Viewer, the API or the language bindings.
+   - Added support for fuselage walls
+   - Support for fuselage profiles with kinks.
+   - Added support for new symmetry flags `inherit` and `none`. The symmetry flag `inherit` causes a geometric
+     component to inherit the symmetry flag from its parent geometry. This was and still is the default behavior in
+     TiGL. The new symmetry type `none` allows the addition of components to mirrored geometries without respecting the
+     symmetry of the parent.
+   - Substantially improve performance of `::tiglCheckPointInside`.
+   - Reduce complexity of the fuselage geometry: The profiles of the fuselage are reparametrized to create simpler
+     B-Spline surface. This will improve robustness of boolean operations and increase the general performance at the
+     cost of a small error in the geometry creation, that should be negligable in most cases.
+
+ - New API functions:
+
+   - Added new api function ``::tiglComponentGetType``
+   - For control devices, the `deflection` value has been renamed to `control_parameter`. Therefore the functions
+     - `::tiglControlSurfaceGetMinimumDeflection`
+     - `::tiglControlSurfaceGetMaximumDeflection`
+     - `::tiglControlSurfaceGetDeflection`
+     - `::tiglControlSurfaceSetDeflection`
+     are marked as deprecated. Please use the new functions
+     - `::tiglControlSurfaceGetMinimumControlParameter`
+     - `::tiglControlSurfaceGetMaximumControlParameter`
+     - `::tiglControlSurfaceGetControlParameter`
+     - `::tiglControlSurfaceSetControlParameter`
+     instead.
+
+
+ - Fixes:
+
+   - Fixes compilation errors with MinGW and VS 2019
+   - Fixes compilation errors with qt4
+   - Fixes compilation errors with OpenCascade 7.4.0 and OCCT 7.2
+   - Several fixes to internal python bindings
+   - Several fixes to control devices code.
+   - Fixes invalid segment meta data in VTK export
+   - fixing problem in ReadCPACS when using rotors (invalidation during read of segments caused error in lazy creation of attached rotor blades)
+   - generating classes for genericSystems
```

Comment:
Is this a fix? I would have completely ommited this, but maybe it deserves to be in the changelog.

Reply:
I missed that one!